libraries/libfc/include/fc/crypto/bls_private_key.hpp

```diff
@@ -6,47 +6,46 @@
 namespace fc::crypto::blslib {
 
    namespace config {
-      constexpr const char* bls_private_key_base_prefix = "PVT";
-      constexpr const char* bls_private_key_prefix = "BLS";
+      const std::string bls_private_key_prefix = "PVT_BLS_";
    };
 
    class bls_private_key
    {
       public:
+
          bls_private_key() = default;
          bls_private_key( bls_private_key&& ) = default;
          bls_private_key( const bls_private_key& ) = default;
-         bls_private_key& operator=( const bls_private_key& ) = default;
-         explicit bls_private_key( std::vector<uint8_t> seed ) {
-            _seed = std::move(seed);
+         explicit bls_private_key(const std::vector<uint8_t>& seed ) {
+            _sk = bls12_381::secret_key(seed);
          }
+         explicit bls_private_key(const std::string& base58str);
 
-         // serialize to/from string
-         // TODO: determine format to use for string of private key
-         explicit bls_private_key(const string& base58str);
-         std::string to_string(const fc::yield_function_t& yield = fc::yield_function_t()) const;
+         bls_private_key& operator=( const bls_private_key& ) = default;
+
+         std::string to_string(const yield_function_t& yield = yield_function_t()) const;
 
          bls_public_key     get_public_key() const;
-         bls_signature      sign( const vector<uint8_t>& message ) const;
+         bls_signature      sign( const std::vector<uint8_t>& message ) const;
 
          static bls_private_key generate();
 
-         static bls_private_key regenerate( vector<uint8_t> seed ) {
+         static bls_private_key regenerate( std::vector<uint8_t> seed ) {

```

Comment:
Small detail, but I don't get the name, and I don't see why it is needed since we have the  constructor? Why not call the constructor directly?